runelite-client/src/main/java/net/runelite/client/plugins/ExternalPluginManager.java

```diff
@@ -595,8 +206,10 @@ public void startExternalPluginManager()
 
 				startExternalPluginManager();
 			}
-
-			log.error("{}", ex.getMessage());
+			else
+			{
+				log.error("Could not load plugins", ex);
```

Comment:
The other branch already logs & deals with that exception type, so no need to log twice(?)